File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q27.py
def main():
    #Input
    value_seconds = int(input("Enter a value in seconds: "))

    #Processing
    hours = round(value_seconds / 3600)
    minutes = round((value_seconds % 3600) / 60)
    seconds = (value_seconds % 3600) % 60

    #Output
    print(f"The entered value of {value_seconds} converted in hours, minutes and seconds is equal to: {hours} hours, {minutes} minutes and {seconds} seconds.")

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q27.js
import { question } from "readline-sync";
function main(){
    //Input
    const seconds = parseInt(question("Enter a value in seconds: "))

    //Processing
    const value_hours = Math.floor(seconds / 3600)
    const value_minutes = Math.floor((seconds % 3600) / 60)
    const value_seconds = (seconds % 3600) % 60

    //Output
    console.log(`The entered value of ${seconds} seconds converted to hours, minutes and seconds is equal to: ${value_hours} hours, ${value_minutes} and ${value_seconds} seconds`)

}
    



main()